chat.js

```diff
@@ -948,3 +948,409 @@ Chat.toDurationString = function (number, options) {
 	}
 	return parts.slice(positiveIndex).reverse().map((value, index) => value ? value + " " + unitNames[index] + (value > 1 ? "s" : "") : "").reverse().slice(0, precision).join(" ").trim();
 };
+
+Chat.getPokemonIcon = function (pokemon, facingLeft) {
```

Comment:
Well, like urkerab did, one can modify pokedex.js and make a new `template.spritenum` property.